objects/src/transaction/proven_tx.rs

```diff
@@ -20,7 +19,7 @@ pub struct ProvenTransaction {
     account_id: AccountId,
     initial_account_hash: Digest,
     final_account_hash: Digest,
-    consumed_notes: Vec<ConsumedNoteInfo>,
+    consumed_notes: Vec<Nullifier>,
```

Comment:
I understand that `consumed_notes` are defined by their nullifiers. But why don't we just name the variable as is, as `nullifiers`

Reply:
I think nullifiers would not be descriptive enough. We could use `consumed_note_nullifiers` but that's a but too long.